### ISSUE TITLE
Unable to restart apache2

### ISSUE THREAD

**Issue by mortuzahossain** [edgarrc/MonoUbuntu#2]
Unable to restart 
```
Restarting apache2 (via systemctl): apache2.serviceJob for apache2.service failed because the control process exited with error code.
See "systemctl status apache2.service" and "journalctl -xe" for details.
 failed!
```
also can not restart apache again:
```
/etc/init.d/apache2 restart 
```
your help is highly needed.

**Comment by edgarrc** [comment]
@mortuzahossain What is the ubuntu version that you are running?

**Comment by mortuzahossain** [comment]
I try with Ubuntu 14.04LTS, 16.04LTS,20.04LTS desktop, and 16.04 server but failed every time.
I am using an HP notebook.  I also try to install them into a virtual box.

**Comment by edgarrc** [comment]
Ok I will create a 16.04 (64bit) VM to test.

**Comment by mortuzahossain** [comment]
Thanks a lot, and your help is highly appreciable. Thank you so much.

**Comment by edgarrc** [comment]
Looks like the mod-mono for the latest release of apache on 16.04 is broken. 

Instead of trying to fix the 16.04, I changed the script to work on Ubuntu 20.04 Server 64bit. This project is focused for production, so I have not tested on 20.04 desktop, but It probably will work on desktop too.

Just run the script on a new installation of 20.04
